core/src/jvmMain/kotlin/com/ustadmobile/core/util/ShrinkUtils.kt

```diff
@@ -81,6 +82,27 @@ object ShrinkUtils {
     }
 
 
+    /**
+     * Validate a ratio string that should be in the form of "x:y" where x and y are positive,
+     * non-zero integers. ffprobe might return N/A or something other than a valid aspect ratio,
+     * so this input needs validated.
+     *
+     * @return the ratioStr trimmed if it is valid, null otherwise
+     */
+    fun validateRatio(ratioStr: String): String? {
```

Comment:
This should be internal (it is accessed by test functions, but it is not really part of any API.